Governance/tech/conventions.md

```diff
@@ -1,147 +1,295 @@
 ```
 
-```{dropdown} cell_methods 
-CF -
+````
+
+```{dropdown} **cell_methods** 
+**CF - recommended**<br>
+[`cell_methods`](http://cfconventions.org/cf-conventions/cf-conventions.html#appendix-cell-methods) describes a characteristic (usually statistical) of the cell values. `cell_methods` is useful where a variable could be intepreted both as an extensive (which depends on the size of the cell) or intensive (which does not) quantity.
+For example, a precipitation rate could be instantaneous (intensive) or a mean value over the time interval specified by the cell (extensive).<br>
+
+**Requirements**<br>
+* It is a string attribute comprising a list of `name: method` pairs, where `name` indicates the dimension or coordinate to which the method is applied. For example, `cell_methods="time: mean"` means that each cell value represents the  time averaged value.
+* To list different methods for different axis: `cell_methods="time: mean longitude: max"`.
+* The values of method should be selected from [this list](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#appendix-cell-methods).
+
+**Recommendations**<br>
+* The word 'area' can also be used as `name` (see [here](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#cell-methods-no-coordinates)).
+* To clarify that a cell value is "instantaneous" use `point` as method.
+* Every data variable should include for each of its dimensions and each of its scalar coordinate variables the cell_methods information of interest (unless this information would not be meaningful). 
+* It is especially recommended that cell_methods be explicitly specified for each spatio-temporal dimension and each spatio-temporal scalar coordinate variable.
 ```
 
-```{dropdown} missing and valid data related 
-CF -
-_FillValue, missing_value, valid_min, valid_max, and valid_range attributes to indicate missing data. Missing data is allowed in data variables and auxiliary coordinate variables. Generic applications should treat the data as missing where any auxiliary coordinate variables have missing values; special-purpose applications might be able to make use of the data. Missing data is not allowed in coordinate variables.
+```{dropdown} **missing and valid data** 
+**CF - recommended**<br>
+[`_FillValue`, `missing_value`, `valid_min`, `valid_max`, and `valid_range`](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#missing-data) attributes are used to indicate missing data. Missing data is allowed in data variables and auxiliary coordinate variables. Generic applications should treat the data as missing where any auxiliary coordinate variables have missing values; special-purpose applications might be able to make use of the data.<br>
+Missing data is not allowed in coordinate variables.<br>
+
+**-FillValue**<br>
```

Comment:
I would argue that it's very important to define this value so that NaNs are properly interpreted by as many tools as possible.

Reply:
Is this better?
Generic applications should treat the data as missing where any auxiliary coordinate variables have missing values; it's very important to define this value so that NaNs are properly interpreted by as many tools as possible.<br>

Reply:
Yep sounds good